src/java/us/kbase/templates/module_makefile.vm.properties

```diff
@@ -120,6 +121,7 @@ build-test-script:
 	echo 'script_dir=$$(dirname "$$(readlink -f "$$0")")' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
 	echo 'export KB_DEPLOYMENT_CONFIG=$$script_dir/../deploy.cfg' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
 	echo 'export KB_AUTH_TOKEN=`cat /kb/module/work/token`' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
+	echo 'rm -rf $(WORK_DIR)' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
```

Comment:
Should this go after the command to run tests? Or do you only want to delete the files before you run tests? I thought maybe it would delete the files after the tests run so the user isn't left with anything sitting in the tmp dir.

Reply:
I tried out another variation that adds the following few lines to the end of the `test` target (line 146):
```sh
echo 'Removing temp files...'
rm -rf $(WORK_DIR)
echo '...done removing temp files.'
```

Just so it's very explicit when exactly it happens. And it might be very slightly safter to set `WORK_DIR` to an absolute path like `/kb/module/work/tmp` instead of relative

Reply:
So I've often used the tempfiles for debugging failed tests (or just understanding what kind of output wrapped tools are producing) so I wouldn't want to remove the files after running the tests by default. I could make the the test script a bit more explicit though as you recommend.